property-service/src/main/java/com/realtybuy/propertyservice/model/Property.java

```diff
@@ -0,0 +1,26 @@
+package com.realtybuy.propertyservice.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.math.BigDecimal;
+
+@Document(value = "property")
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Data
+public class Property {
+    @Id
+    private String id;
+    private String name;
+    private String description;
+    private String address;
```

Comment:
for this member address we can use in string format, but in the next version we can implement a class with its own address atributes.